nss-day-cohort-48/rare-client-spicy-sauce-scouts
# author profile

#### Changes Made
1.Made changes to author files
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout tlc-author-profile
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    Test to see if you can click on the author name from the posts section and get to the author page.
4. View code file. - check the code, see if it makes sense.
